pages/api/reservoir/[...slug].ts

```diff
@@ -33,6 +34,25 @@ const proxy = async (req: NextApiRequest, res: NextApiResponse) => {
   setParams(url, query)
 
   if (endpoint.includes('redirect/')) {
+    // Redirect eth and weth currency icons to self-hosted
+    // versions without any padding
+    endpoint = endpoint.toLowerCase()
+
+    if (
+      (chainPrefix == 'ethereum' || chainPrefix == 'goerli') &&
```

Comment:
can we potentially just change this to be something more like this:
`[mainnet.id, goerli.id].includes(chain.id)`
Feels cleaner.